SpatialGDK/Build/Scripts/BuildWorker.bat

```diff
@@ -1,22 +1,63 @@
 @echo off
 
-rem Generate worker configs
-pushd "%~dp0..\..\..\..\..\..\spatial"
+rem Input args: BuildName, Platform, BuildConfiguration, uproject 
+
+rem If we are running as a project plugin then this will be the path to the spatial directory.
+:BuildAsProjectPlugin
+if exist "%~dp0..\..\..\..\..\..\spatial" (

```

Comment:
If the engine root is the directory just above the project root, won't this relative path be the same for both project plugin and the engine plugin?

Reply:
This is unfortunately not true in all cases. You can use an Engine plugin but with the project located in any directory.

Reply:
What I mean is that the relative path `..\..\..\..\..\..\spatial` resolving to a valid directory does not necessarily confirm that we have a project plugin.